data/templates/SystemInvokerTest.csx

```diff
@@ -0,0 +1,18 @@
+﻿private static string GetQuery(Dictionary<string, dynamic> cxt)
+{
+    return
+    $@"cluster('MockCluster).database('Mockdb').DiagnosticRole
```

Comment:
Instead of putting mock table here, since this is a template, use <YOUR_KUSTO_QUERY> here.
And in test, replace this string with your mock query.